code/_onclick/hud/fullscreen.dm

```diff
@@ -149,7 +149,7 @@
 	screen_loc = "WEST,SOUTH to EAST,NORTH"
 	icon_state = "flash"
 	plane = SPLASHSCREEN_PLANE
-	layer = SPLASHSCREEN_LAYER - 1
+	layer = -1
```

Comment:
Uh? Shouldn't this be a define?

Reply:
It probably would be clearer as a define, sure.